hull-vidispine-addon/hull-vidispine-addon.yaml

```diff
@@ -1253,28 +1493,24 @@ hull:
                                 $this.WriteLog("**** '$entityType' entry '$identifier' already exists, not overwriting due to flag 'overwriteExisting' not being set")
                             }
                         }
-                        elseif ($value.noGet -or ($statusCode -eq 404))
+                        elseif ($entity.noGet -or ($statusCode -eq 404))
                         {
```

Comment:
It seems that here is the "root key" of the client used and not the value provided in the "root.config.identifier"